### DISCUSSION TITLE
good article by @sugar-cat7

### DISCUSSION THREAD

**Discussion by ryoppippi** [samchon/typia#1361] [Show and tell]
I found a good Japanese article on logging using `Typia` by @sugar-cat7 !!

https://zenn.dev/king/articles/707a59953a1c95

**Comment by samchon** [comment]
@{number=1361; body=Thanks for sharing this article. Acknowledged as a community/show-and-tell post; no repository change is needed from this discussion, so I am closing it as answered.}.body
